scripts/bin/fks-moveToFykos.sh

```diff
@@ -47,14 +47,14 @@ if [ `git status | grep -E 'Untracked|Changes' | wc -l` != 0 ]; then
 fi
 
 file=`sed "s/[J|N|P|E|S][-]\([a-z|_]*\)/\1/g" <<< $3`
-file="?-R*S*-?-$file.tex"
+file="?-R*S*-*-$file.tex"
```

Comment:
I didn't find any proper definition of `*` expansion. What about
`"?-R*S*$file.tex"`?

Reply:
Or `"?-R*S*-$file.tex"`?